src/components/SecondaryPanes/Frames.js

```diff
@@ -6,6 +6,8 @@ import { connect } from "react-redux";
 import actions from "../../actions";
 import { endTruncateStr } from "../../utils/utils";
 import { getFilename } from "../../utils/source";
+const get = require("lodash/get");
+const zip = require("lodash/zip");
```

Comment:
Can we change these two lines to es import like `import get from "lodash/get"` ?